src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -15,9 +20,13 @@ public class CommandBox extends UiPart<Region> {
 
     public static final String ERROR_STYLE_CLASS = "error";
     private static final String FXML = "CommandBox.fxml";
+    private static final int ZERO_INDEX = 0;
```

Comment:
Good coding standard practiced here! 

We should eliminate the use of 'magic numbers' to improve coding readability!